docs/ai.md

```diff
@@ -45,10 +47,38 @@ services:
                 - gpu
 ```
 
+```yaml
+# podman-compose
+---
+services:
+  ollama:
+    image: ollama/ollama
+    container_name: ollama
+    restart: unless-stopped
+    ports:
+      - 11434:11434
+    volumes:
+      - ./ollama_v:/root/.ollama:z
+    devices:
+      - nvidia.com/gpu=all
```

Comment:
This might work with docker-compose as well. If so we can probably unify the 2 blocks (`:z` worked with docker-compose)

Reply:
Does not work for `docker`

```
Error response from daemon: could not select device driver "cdi" with capabilities: []
```